homeassistant/components/restore_state.py

```diff
@@ -0,0 +1,72 @@
+"""
```

Comment:
We can remove this file. Restoration is now performed without an extra component.

We also don't need include/exclude options because platforms will know if they support restoration or not.